templates/user/user-base.jade

```diff
@@ -49,7 +49,7 @@ block body
                 b {% trans "Total points:" %}
                 |  #{user.points}
                 br
-                .pp-feature(style="display: none")
+                .pp-feature(style=(None if perms.judge.test_site else 'display: none'))
```

Comment:
I do not think that's valid. These are dictionary lookups, and when PyJade handles them, you are to write `perms['judge']['test_site']`.